### ISSUE TITLE
Too many authors causes iacrcc.cls to bomb

### ISSUE THREAD

**Issue by kmccurley** [IACR/latex#120]
It appears that as soon as 27 authors are added with \addauthor, we get:
```
! Missing \endcsname inserted.
<to be read again> 
\protect 
l.43 \addauthor[inst={24,56}]{Jean-Sébastien Coron}
```

**Comment by kmccurley** [comment]
I believe this is caused by line 556, because if I comment this out it works:
``
\expandafter\let\csname IACR@author@footnote\@Alph\num@authors \endcsname\IACRfootnote% Copy the pgf key
```

**Comment by jwbos** [comment]
Fixed in 40c80e4092ab51009f28c006d6f0034d60a3b4c5.
